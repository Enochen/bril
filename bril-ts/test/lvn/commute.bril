@main {
  a: int = const 4;
  b: int = const 2;
  sum1: int = add a b;
  sum2: int = add b a;
  prod1: int = mul sum1 sum2;
  prod1: int = mul a b;
  prod2: int = mul sum2 sum1;
  prod1: int = mul sum1 sum2;
  print prod1;
}
