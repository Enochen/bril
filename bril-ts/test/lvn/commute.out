@main {
  lvn_temp_0: int = const 36;
  print lvn_temp_0;
}
